nbconvert/filters/tests/test_markdown.py

```diff
@@ -118,8 +118,8 @@ def test_markdown2html_heading_anchors(self):
         ]:
             self._try_markdown(markdown2html, md, tokens)
 
-    def test_markdown2html_math(self):
-        # Mathematical expressions should be passed through unaltered
+    def test_markdown2html_math_noescape(self):
+        # Mathematical expressions not containing <, >, & should be passed through unaltered
```

Comment:
Unifying them is probably ideal, but I'd merge a PR if that was the only thing outstanding.